docs/getting-started/all-dcrctl-rpc-commands.md

```diff
@@ -0,0 +1,162 @@
+# <img class="dcr-icon" src="/img/dcr-icons/Dcrtl.svg" /> All `dcrctl` RPC commands
```

Comment:
1. Can "All" be omitted? Same for lines 5 & 85, and the filename.
2. The term "RPC methods" is used below, it would be better to stay consistent.
3. Please use title case (e.g. "`dcrctl` RPC Commands") for consistency with other pages.